care/facility/api/serializers/shifting.py

```diff
@@ -182,16 +244,25 @@ def create(self, validated_data):
 
         validated_data["is_kasp"] = False
 
-        orgin_facility_external_id = validated_data.pop("orgin_facility")["external_id"]
+        # orgin_facility_external_id = validated_data.pop("orgin_facility")["external_id"]
```

Comment:
Why is this commented out?

Reply:
Unused variable - Linter change. orgin_facility_external_id is not used anywhere. (L248 is the only place and it was already commented).